docs/2023/BeckN/Decentralized Health Protocol/updates/2023-08-04.md

```diff
@@ -4,13 +4,12 @@ author: Karthik S
 ---
 
 ## Milestones
-- [ ] Give the description about Milestone 1
-- [ ] Give the description about Milestone 2
-- [ ] Give the description about Milestone 3
-- [ ] Give the description about Milestone 4
```

Comment:
Please add milestones

Reply:
Ma'am, These are the milestones achieved this week:
- [x] Documented new use cases as suggested by Super mentor
- [x] Added example jsons
- [x] Worked on adding on_search examples for DSEP